src/slides/slides.css

```
@@ -199,26 +200,21 @@ body {
 .reveal pre {
   margin: var(--content-block-vertical-margin) 0;
   box-shadow: none;
+  border: 1px solid #ddd;
```

```suggestion
  border: 1px solid #d9d9d9;
```

According to the [guidelines](https://docs.google.com/presentation/d/1MIXNnkLNSfOgdlOnK5Fjqkv787OpXj4_BXAzKE7Swqk/edit#slide=id.g6feffcbd02_0_351), slide 16.